engine/spec/components/citizens_advice_components/disclosure_spec.rb

```diff
@@ -72,7 +72,7 @@
       ).with_content("Example content")
     end
 
-    it { is_expected.to have_css "[data-toggle-target-id='my-id']" }
+    it { is_expected.to have_css "[aria-controls='my-id']" }
```

Comment:
This has been switched out as the component no longer needs `data-toggle-target-id` to work. Testing against aria attributes is preferable too.